.ocamlformat

```diff
@@ -1,4 +1,4 @@
-version=0.18.0
+version=0.19.0
```

Comment:
Is this change really needed? if it is, move it to a separate PR.

Reply:
#679